tizen-api/src/tizen-api-pipeline.c

```diff
@@ -36,6 +36,41 @@
 
 #define DLOG_TAG "nnstreamer-capi-pipeline"
 
+#define handle_init(type, name, h) \
+  nns_##type *name= (h); \
+  nns_pipeline *p; \
+  element *elem; \
+  int ret = NNS_ERROR_NONE; \
+  if (h == NULL) { \
+    dlog_print (DLOG_ERROR, DLOG_TAG, \
+        "The handle to be unregistered is invalid"); \
```

Comment:
plz update common log message (to be unregistered)

Reply:
Fixed!